### ISSUE TITLE
Please can You explain me how can I notice me about changing the viewController?

### ISSUE THREAD

**Issue by SergeyOleynich** [StefanLage/SLPagingView#7]
I'm new in iOS development. I see this in .h
/*
-  Block delegates
  */
  typedef void(^SLPagingViewMoving)(NSArray *subviews);
  typedef void(^SLPagingViewMovingRedefine)(UIScrollView \* scrollView, NSArray *subviews);
  typedef void(^SLPagingViewDidChanged)(NSInteger currentPage);

Please can You explain me how I can notice me about changing the viewController using this blocks?

I've implement this in my appDelegate:

```
pageViewController.didChangedPage = ^(NSInteger interr) {
    NSLog(@"SOMETHING");

    [[NSNotificationCenter defaultCenter] postNotificationName:@"Sergey" object:nil userInfo:nil];
    NSLog(@"%i", interr);

};
```

(use this in demo app in appDelegate). 
I've implement notification and addObserver in another controller but nothing happen. And all three controllers dealloc.
And how can I call viewDidAppear from my controller.
And also I notice that this block return me number 1 and 2. And did't return the number of my first controller. Is it the bug?

How can I say my controller that it is now display on the screen and it should be update?

**Comment by StefanLage** [comment]
Actually the block **didChangePage** was called only if an **UIPageControl** was initialized.
So you shouldn't be notify about the update.
Anyway there was a bug, I fixed it! bf855082b31a0f5cebcfc77b4d8332f0cb291124
Now you can use it although there is no **UIPageControl.**

**Comment by SergeyOleynich** [comment]
Thanks)
